### DISCUSSION TITLE
Ingest local prometheus data to Openpbserve

### DISCUSSION THREAD

**Discussion by Angel0r** [openobserve/openobserve#1814] [Q&A]
Hey!

I am experimenting with OpenObserve. I have been using for a long time Prometheus-stack and I want to test and get familiarised with OpenObserve.
I am running OpenObserve in my local Kubernetes cluster.
I have downloaded Prometheus data from our production Kubernetes cluster and I want to ingest the data into openobserve and test it. Is there any way to do that?

/Angelos

**Comment by prabhatsharma** [comment]
I am not sure if there is a tool that can export data directly from tsdb files of prometheus. You could however ingest metrics to OpenObserve using Prometheus remote write or via json - https://openobserve.ai/docs/api/ingestion/metrics/json/

**Comment by Angel0r** [comment]
Hey @prabhatsharma!

Thanks for the answer. 
I have tried both remote write and JSON but then again those ways are to connect with prometheus and ingest real-time data.

I want to use a sample of prometheus data so I can benchmark it.

/Angelos

**Reply by prabhatsharma** [reply]
Just to be clear. Is this your question? - `How do I import prometheus tsdb data into OpenObserve?`

**Reply by Angel0r** [reply]
Yes but not in real time. The data is already downloaded locally. In short, I want to migrate the all data from my prometheus cluster into OpenObserve.
